packages/cerebral-provider-forms/index.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./lib')
```

Comment:
it's already covered with main property in package.json. No need to have extra file